ophyd/epics_motor.py

```diff
@@ -281,64 +281,13 @@ def report(self):
         rep['pv'] = self.user_readback.pvname
         return rep
 
-    @property
-    def hints(self):
-        """Provide hints to bluesky
-
-        The default value is ::
-
-           {'fields': [self.user_readback.name]}
-
-        To override this, set another dictionary.
-
-        To restore the default value set ``None``
-
-        To suppress all hints set ``{}``
-        """
-        if self._hints is None:
-            return {'fields': [self.user_readback.name]}
-        return self._hints
-
-    @hints.setter
-    def hints(self, val):
-        self._hints = val if val is None else dict(val)
-
 
 class MotorBundle(Device):
     """Sub-class this to device a bundle of motors
 
     This provides better default behavior for ``hints``,
     ``read_attrs`` and ``configuration_attrs``
     """
-    _hints = None
```

Comment:
MotorBundle merges the hints of its child motors. This behavior is still desirable and different from the behavior that has been moved up to `Device`. This block should not be deleted.